pommerman/network/ionc/__init__.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+"""IonClient Entry point"""
+
+# Global Imports
+import ui
+from .constants import LangEN as Strings
+from .network import Network as net
+from signal import signal, SIGINT
+from gym.spaces import Discrete
+
+
+# Functions
+def init():
+    """Initialize the application"""
+
+    def exit_handler(_s, _h):
+        """Handle exiting the application"""
+        ui.fatal(ui.yellow, "Exiting..")
+
+    signal(SIGINT, exit_handler)
+    if ui.ask_yes_no(Strings.server_prompt.value):
+        domain = ui.ask_string(Strings.server_domain.value)
+        if domain is None:
+            ui.fatal(Strings.e_invalid_domain.value)
+    else:
+        domain = "play.pommerman.com:5050"
+    ui.info(Strings.server_connecting_p1.value, ui.yellow,
+            Strings.server_connecting_p2.value, ui.reset,
+            Strings.server_connecting_p3.value)
+    network = net(domain)
+    status = network.check()
+    if not status[0]:
+        ui.fatal(Strings[status[1]].value)
+    else:
+        ui.info(Strings.server_connected.value, ui.yellow,
+                Strings.server_players.value,
+                str(status[1]) + ",", Strings.server_matches.value, status[2])
+    intent(network)
+
+
+def intent(network):
```

Comment:
Is this function only intended to be called from this module? If so, make that clear by making it private.

Reply:
Actually, it can be called from anywhere as I rewrote the client a tad bit so it doesn't depend on global variables. If you feed it a valid network object it'll work as it would. I'm considering making the UI completely optional and have a programmatic way to use this.